### PR TITLE
chore: toggle on sdo (FPLA-3152)

### PR DESCRIPTION
**Pull request by toireasam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-3152

### Change description ###
Toggle on SDO stories
https://tools.hmcts.net/jira/browse/FPLA-2627

https://tools.hmcts.net/jira/browse/FPLA-2813

https://tools.hmcts.net/jira/browse/FPLA-3067

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: ccd-definition/CaseEvent/CareSupervision/MultiState.json [hunk 1]
```diff
@@ -770,5 +770,22 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/remove-managing-organisation/submitted",
     "EventEnablingCondition": "outsourcingPolicy.Organisation.OrganisationID != \"\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addGatekeepingOrder",
+    "Name": "V2: Add the gatekeeping order",
```
Comment (joez-f) [inline ccd-definition/CaseEvent/CareSupervision/MultiState.json:778]: Need to remove V2 from the name here

Comment (toireasam) [reply ccd-definition/CaseEvent/CareSupervision/MultiState.json:778]: will do :) wasn't sure as it was like this in protoype

File: ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json [hunk 2]
```diff
@@ -416,5 +416,27 @@
...
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "draftSDO",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-publiclaw-gatekeeper",
+          "caseworker-publiclaw-judiciary"
+        ],
+        "CRUD": "CR"
```
Comment (joez-f) [inline ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json:430]: Needs to be just R, don't want them to use old event

Comment (joez-f) [reply ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json:430]: i.e move them into the section down below

Comment (toireasam) [reply ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json]: oh very good point haha!

**Review by joez-f — APPROVED** [review]
LGTM

**Review by samuelzerozero — APPROVED** [review]

**Pull request merged**